app/views/account.py

```diff
@@ -370,7 +475,14 @@ def get(self, account_type_id):
                                          message=err.message,
                                          success=False,
                                          status=HTTPStatus.NOT_FOUND)
-            return response.error_response()
+        except Exception as err:
+            logger.exception(err)
+            response = ResponseGenerator(data={},
+                                         message=err,
+                                         success=False,
+                                         status=HTTPStatus.NOT_FOUND)
```

Comment:
Status should not be NOT_FOUND in case of generic Exception.It is applicable to all places where you have used NOT_FOUND in Exception.

Reply:
Made the changes as suggested.